### DISCUSSION TITLE
Unable to access some camunda APIs after switching from formsflow version 4.0.5 to 5.0

### DISCUSSION THREAD

**Discussion by ansulalu-aot** [AOT-Technologies/forms-flow-ai#1321] [Q&A]
I was previously using version 4.0.5 of formsflow.ai , but after switching to version 5.0, I am unable to access Camunda APIs like /camunda/engine-rest/process-instance.

**Comment by yaseen-aot** [comment]
Yes , you can't able to access some Camunda API's , because from v5 onwards we only expose some of Camunda API's which are used by Formsflow.

**Reply by ansulalu-aot** [reply]
Can you provide which are the APIs exposed ?

**Reply by yaseen-aot** [reply]
Before formsflow v5 , we used " http://{ip}:8000/camunda/engine-rest " as  camunda API base url . From v5 onwards , we are using  "http://{ip}:8000/camunda/engine-rest-ext/v1/ " as  camunda API base url . Here is the list of camunda API 's exposed :
[Camunda API’s exposed in formsflow (1).pdf](https://github.com/AOT-Technologies/forms-flow-ai/files/11050304/Camunda.API.s.exposed.in.formsflow.1.pdf)
